server.go

```diff
@@ -650,12 +650,14 @@ func newServer(listenAddrs []net.Addr, chanDB *channeldb.DB, cc *chainControl,
 		chanGraph, selfNode, queryBandwidth,
 	)
 
+	paymentControl := channeldb.NewPaymentControl(chanDB)
+
 	s.chanRouter, err = routing.New(routing.Config{
 		Graph:              chanGraph,
 		Chain:              cc.chainIO,
 		ChainView:          cc.chainView,
 		Payer:              s.htlcSwitch,
-		Control:            channeldb.NewPaymentControl(chanDB),
+		Control:            routing.NewControlTower(paymentControl),
```

Comment:
nit: looks like should be in the "move control tower" commit?

Reply:
yes, that's right